packages/codegen-ui-react/lib/__tests__/react-theme-studio-template-renderer.test.ts

```diff
@@ -47,5 +47,14 @@ describe('react theme renderer tests', () => {
       );
       expect(rendererFactory.buildRenderer(loadSchemaFromJSONFile('theme')).renderThemeJson()).toMatchSnapshot();
     });
+
+    it('should render theme json with breakpoints correctly', () => {
+      const rendererFactory = new StudioTemplateRendererFactory(
+        (theme: StudioTheme) => new ReactThemeStudioTemplateRenderer(theme, {}),
+      );
+      expect(
+        rendererFactory.buildRenderer(loadSchemaFromJSONFile('themeWithBreakpoints')).renderThemeJson(),
+      ).toMatchSnapshot();
```

Comment:
an alternative here to snapshots could be loading the stringified json and load into returning the theme object and run expects on the returned theme shape so we know that breakpoints will contain numbers for values
```suggestion
const themeJson = rendererFactory.buildRenderer(loadSchemaFromJSONFile('themeWithBreakpoints')).renderThemeJson()
const themeObject = new Function(`return ${themeJson}`)();
expect(themeObject).toBeDefined(); // or expect other values to exist in the themeObject
```